origami.json

```diff
@@ -18,18 +19,30 @@
             "expanded": true,
             "description": "Standard card",
             "template": "demos/src/main.mustache",
-            "path": "/demos/standard.html"
+            "data": {
+                "card-image-url": "",
+                "card-image-alignment": "",
```

Comment:
Rather than leaving them blank, I wouldn't even add them to the config, they're already big enough. But that's just my taste :P